packages/cli/src/workflows/workflows.controller.ts

```diff
@@ -355,6 +356,26 @@ workflowsController.patch(
 			);
 		}
 
+		const numberOfUsers = await Db.collections.User.count();
+
+		if (numberOfUsers > 1 && !forceSave) {
+			const lastKnownDate = new Date(req.body.updatedAt).getTime();
```

Comment:
Instead of parsing both of these as dates, can't we simply compare them as strings? We have to make sure that FE never changes those (whatever string it gets from BE is sent back as is).

Reply:
`req.body.updatedAt` is a date string but `shared.workflow.updatedAt` is a date object. I found it simpler to compare with `getTime()` rather than to stringify the date object in the exact format of the incoming string date. `toString()` and `toUTCString()` give out a different format, and date objects cannot be compared directly.

Reply:
> We have to make sure that FE never changes those (whatever string it gets from BE is sent back as is).

Added a warning. Not sure how to enforce this client-side though. If they ever mutate and manually test, they will start seeing this error.